src/test/java/org/jabref/logic/formatter/FormatterTest.java

```diff
@@ -38,102 +38,101 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-public class FormatterTest {
+class FormatterTest {
 
     private static ProtectedTermsLoader protectedTermsLoader;
 
     @BeforeAll
-    public static void setUp() {
+    static void setUp() {
         protectedTermsLoader = new ProtectedTermsLoader(
                 new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(),
                         Collections.emptyList(), Collections.emptyList()));
 
+    }
+
+    private static Stream<Formatter> getFormatters() {
+        // all classes implementing {@link net.sf.jabref.model.cleanup.Formatter}
+        // sorted alphabetically
+        // Alternative: Use reflection - https://github.com/ronmamo/reflections
+        // @formatter:off
+       return Stream.of(
+                new CapitalizeFormatter(),
+                new ClearFormatter(),
+                new HtmlToLatexFormatter(),
+                new HtmlToUnicodeFormatter(),
+                new IdentityFormatter(),
+                new LatexCleanupFormatter(),
+                new LatexToUnicodeFormatter(),
+                new LowerCaseFormatter(),
+                new MinifyNameListFormatter(),
+                new NormalizeDateFormatter(),
+                new NormalizeMonthFormatter(),
+                new NormalizeNamesFormatter(),
+                new NormalizePagesFormatter(),
+                new OrdinalsToSuperscriptFormatter(),
+                new ProtectTermsFormatter(protectedTermsLoader),
+               new RegexFormatter("(\" \",\"-\")"),
```

Comment:
I don't want to do nitpkicking but that Regex Formatter indentation is clear off

Reply:
Yeah, no idea why there were these `@formatter` comments that prevented an automatic code formatting...